tests/unit_tests/utils/command.cpp

```diff
@@ -0,0 +1,57 @@
+#include "utils/command.hpp"
+#include "common/test.hpp"
+
+#include <unistd.h>
+
+using namespace polybar;
+
+TEST(Command, status) {
+  // Test for command<output_policy::IGNORED>::exec(bool);
+  {
+    auto cmd = command_util::make_command<output_policy::IGNORED>("echo polybar > /dev/null");
+    int status = cmd->exec();
+
+    ASSERT_EQ(status, EXIT_SUCCESS);
```

Comment:
Please use `EXPECT_*` everywhere so that we don't see only a part of the test fail. Use `ASSERT_*` if the rest of the code would otherwise have undefined behavior (for example for NULL checks).